week-4/Homework/mandatory/4-groceries.js

```diff
@@ -39,6 +41,19 @@ Exercise 2:
 */
 // Gather weekend item names into this array
 let weekendGroceriesToBuy = [];
+ for(let weekendDay in weeklyMealPlan){
+  if(weekendDay === 'saturday' || weekendDay === 'sunday'){
+  
+    for(let i = 0; i < weeklyMealPlan[weekendDay].length; i++){
```

Comment:
This is a good solution - well done. You can do someting similar to this with the previous exercise too